app/views/journeys/expenses/construction/ConstructionIndustryAmountView.scala.html

```diff
@@ -0,0 +1,67 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import viewmodels.InputWidth._
+@import models.common.BusinessId
+@import models.common.TaxYear
+@import models.common.UserType
+@import controllers.journeys.expenses.construction.routes
+
+@this(
+        layout: templates.Layout,
+        formHelper: FormWithCSRF,
+        govukErrorSummary: GovukErrorSummary,
+        govukInput: GovukInput,
+        govukButton: GovukButton
+)
+
+@(form: Form[_], mode: Mode, userType: String, taxYear: Int, businessId: String)(implicit request: Request[_], messages: Messages)
+
+@layout(pageTitle = title(form, messages(s"constructionIndustryAmount.heading.$userType"))) {
+
+    @formHelper(action = routes.ConstructionIndustryAmountController.onSubmit(taxYear, businessId, mode), Symbol("autoComplete") -> "off") {
+
+        @if(form.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummaryViewModel(form))
+        }
+
+        @govukInput(
+            InputViewModel(
+                field = form("value"),
+                label = LabelViewModel(HtmlContent(buildLabelContent(userType))).asPageHeading()
+            )
+            .asNumeric()
+            .withWidth(Fixed10)
+            .withPrefix(PrefixOrSuffix(content = Text("£")))
+        )
+
+        @govukButton(
+            ButtonViewModel(messages("site.continue"))
+        )
+    }
+}
+@buildLabelContent(authUserType: String) = {
```

Comment:
Maybe do something like [this](https://github.com/hmrc/income-tax-self-employment-frontend/pull/102/files#diff-f328d2fdf2d7fe2ee51a3b037ebead5efee3871862fe0eace751ee96bbc86e67) so the logic is in order?